Schema/FeatureManagement.v2.0.0.schema.json

```diff
@@ -23,7 +23,7 @@
                     "title": "Feature Flags",
                     "description": "Declares feature flags based on Microsoft Feature Flag schema.",
                     "items": {
-                        "$ref": "https://raw.githubusercontent.com/Azure/AppConfiguration/main/docs/FeatureManagement/FeatureFlag.v2.0.0.schema.json"
+                        "$ref": "https://raw.githubusercontent.com/Microsoft/FeatureManagement/main/Schema/FeatureFlag/FeatureFlag.v2.0.0.schema.json"
```

Comment:
It doesn't look like we have the benefits as you described.

Reply:
How about this?

We will have a 1:1 version map between FeatureManagement schema and feature flag schema.

And whenever we need to put a FeatureManagement schema reference, we will give them the root folder https://github.com/microsoft/FeatureManagement/tree/main/Schema
In this case, customer will always be bring to the latest version.